zilliqa/src/scilla.rs

```diff
@@ -568,9 +570,10 @@ impl StateServer {
                     return Err(err("no active call"));
                 };
 
-                active_call.fetch_blockchain_info(query_name, query_args);
-
-                Ok::<_, ErrorObject>(())
+                match active_call.fetch_blockchain_info(query_name, query_args) {
+                    Ok((success, value)) => Ok(Value::Array(vec![success.into(), value.into()])),
```

Comment:
Small nit: Can we rename `success` to something like `present`? If `fetch_blockchain_info` returns `Ok((false, _))` it **succeeded**, but the block queried wasn't present.